---
 pyproject.toml | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2068516..a26c7ad 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,10 +34,6 @@ pytz = "*"
 PyYAML = ">=5.4.1,<7"
 requests = "^2.31.0"
 MarkupSafe = "^2.0"
-types-beautifulsoup4 = "^4.12.0.20240229"
-types-pyyaml = "^6.0.12.20240311"
-types-pytz = "^2024.1.0.20240417"
-
 
 
 [tool.poetry.scripts]
@@ -51,6 +47,9 @@ pytest = "^6.2.5"
 mypy = "^1.10.0"
 types-requests = "^2.31.0"
 types-pygments = "^2.17.0.20240310"
+types-beautifulsoup4 = "^4.12.0.20240229"
+types-pyyaml = "^6.0.12.20240311"
+types-pytz = "^2024.1.0.20240417"
 
 
 [build-system]
-- 
2.44.0

